### PR TITLE
Create LogEntryData-bigqueryjobcompleted.json

### PR DESCRIPTION
**Pull request by minyu19**
Adding more JSON example CloudEvents

### CODE REVIEW

**Comment by jskeet** [comment]
The files in `testdata` aren't expected to be complete CloudEvents - just the payloads.

File: testdata/google/events/cloud/audit/v1/LogEntryData-bigqueryjobcompleted.json [hunk 1]
```diff
@@ -0,0 +1,92 @@
+{
+  "data": {
```
Comment (grant) [inline testdata/google/events/cloud/audit/v1/LogEntryData-bigqueryjobcompleted.json:2]: As Jon mentioned, put just the _contents_ of the `"data"` property as the JSON object for this file.

**Review by grant — CHANGES_REQUESTED** [review]

File: testdata/google/events/cloud/audit/v1/LogEntryData-bigqueryjobcompleted.json [hunk 2]
```diff
@@ -76,17 +75,4 @@
...
-  "recordedTime": "2021-11-25T21:56:00.276607Z",
-  "resourceName": "projects/my-project/jobs/bqjob_r3ac45813612fa2d6_0000017d591922c9_1",
-  "serviceName": "bigquery.googleapis.com",
-  "source": "//cloudaudit.googleapis.com/projects/my-project/logs/data_access",
-  "specversion": "1.0",
-  "subject": "bigquery.googleapis.com/projects/my-project/jobs/bqjob_r3ac45813612fa2d6_0000017d591922c9_1",
-  "time": "2021-11-25T21:56:00.653866570Z",
-  "type": "google.cloud.audit.log.v1.written"
-}
+  }
```
Comment (grant) [inline testdata/google/events/cloud/audit/v1/LogEntryData-bigqueryjobcompleted.json:78]: Format is off, there should be no space here.

**Review by grant — CHANGES_REQUESTED** [review]

File: testdata/google/events/cloud/audit/v1/LogEntryData-bigqueryjobcompleted.json [hunk 3]
```diff
@@ -0,0 +1,78 @@
+{
+    "insertId": "jpllvgecd7bx",
```
Comment (grant) [inline testdata/google/events/cloud/audit/v1/LogEntryData-bigqueryjobcompleted.json:2]: I can't tell if 2 space indentation is here or more. Should be 2 spaces.

**Review by grant — APPROVED** [review]
Approved

**Review by ace-n — APPROVED** [review]

**Pull request merged**